sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/CryptographyClient.cs

```diff
@@ -250,7 +288,7 @@ public virtual EncryptResult Encrypt(EncryptionAlgorithm algorithm, byte[] plain
         }
 
         /// <summary>
-        /// Decrypts the specified cipher text.
+        /// Decrypts the specified ciphertext.
```

Comment:
nit: Inconsistent with "Decrypts ciphertext". I think this one is more descriptive.

Reply:
See above.